pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -301,9 +301,17 @@ func getPromExemplars[T exemplarType](pt T) []prompb.Exemplar {
 		exemplar := pt.Exemplars().At(i)
 		exemplarRunes := 0
 
-		promExemplar := prompb.Exemplar{
-			Value:     exemplar.DoubleValue(),
-			Timestamp: timestamp.FromTime(exemplar.Timestamp().AsTime()),
+		var promExemplar prompb.Exemplar
+		if exemplar.ValueType() == pmetric.ExemplarValueTypeInt {
```

Comment:
can we use a `switch` on `exemplar.ValueType()` instead of two if-statements?

Reply:
addressed

[d1eb42d](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36688/commits/d1eb42d18c4efeefdb1f03f96c2c5a27b899b917)